src/formBuilder/FormBuilder.test.js

```diff
@@ -263,4 +263,110 @@ describe('FormBuilder', () => {
       .map((error) => error.text());
     expect(errors).toEqual([]);
   });
+
+  it('validates additionalProperties as a valid property', () => {
```

Comment:
It looks like you are including code from another PR.  Please revert these changes.